src/styles/blocks/navigation.scss

```
@@ -0,0 +1,58 @@
+.nav {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: normal;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  justify-content: space-between;
+  padding: 0;
+
+  &__link {
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: #060b35;
+    padding: 23px 0;
+    align-items: center;
+    position: relative;
+    &:hover {
+      color: $link-color;
+    }
+  }
+
+  &__spacing {
+    margin-right: 20px;
+  }
+
+}
+
+.homeLink {
```

Better to do not mix different case styles
```suggestion
.home-link {
```